asset/cloud-client/requirements.txt

```diff
@@ -0,0 +1,2 @@
+google-cloud-storage==1.38.0
```

Comment:
I'm not sure these are valid module versions, I'm using:

```
google-cloud-storage==1.13.0
google-cloud-asset==0.1.1
```

Reply:
There are more things I missed in my initial review.  

You may want to add some entropy to the bucket name in case it doesn't get deleted.  The test seems to be failing the second time it's run for me. An example of adding some randomness to your tests [can be found here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iot/api-client/manager/manager_test.py#L27) where the time is appended to the filename.

I'm seeing the error:

```
TypeError: ('gs://', 'bucket-for-assets2', '/assets-dump.txt') has type tuple, but expected one of: bytes, unicode
``` 

in your test when running on my machine. Are you testing with both Python3 and Python2?

Reply:
The error you're seeing appears to be coming from how you're doing string concatenation:

Replace:
```
-    dump_file_path = "gs://", asset_bucket, "/assets-dump.txt"
```
in your test with
```
+    dump_file_path = 'gs://{}/assets-dump.txt'.format(asset_bucket)
```

You're missing parenthesis on result in `quickstart_exportassets.py`, it should be:

```
print(response.result())
```

In your test file, you're testing for your personal project ID.  This should instead be using the string from earlier:

```
-    assert "uri: \"gs://cai-prober-prod-for-assets/phython-test.txt\"" in out
+    assert dump_file_path in out
```

Anyways, sorry I missed all these before, you caught me sleeping.  Let's get the fixes in tomorrow am.